data.sql

```diff
@@ -14,4 +14,24 @@ VALUES('Agumon', date '2020-02-03', '0', 'true', '10.23'),
 ('Ditto', date '2022-05-14', '4', 'true', '22');
 
 
+INSERT INTO owners(full_name, age)
+VALUES('Sam Smith', '34'),
+('Jennifer Orwell', '19'),
+('Bob', '45'),
+('Melody Pond', '77'),
+('Dean Winchester', '14'),
+('Jodie Whittaker', '38');
+
+
+INSERT INTO species(name)
+VALUES('Pokemon'),
+('Digimon'); 
+
+
+UPDATE animals SET owners_id = '1' WHERE name = 'Agumon';
+UPDATE animals SET owners_id = '2' WHERE name = 'Gabumon' OR name = 'Pikachu';
+UPDATE animals SET owners_id = '3' WHERE name = 'Devimon' OR name = 'Plantmon';
+UPDATE animals SET owners_id = '4' WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+UPDATE animals SET owners_id = '5' WHERE name = 'Angemon' OR name = 'Boarmon';
+
```

Comment:
- [ ] It would be better if we also add the update query for the following:

1. If an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead. Like for example;
```
 UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';
```
Nice job making the `update` query in data.sql file, we have a few to go :man_technologist:

Reply:
Hello, thank you for the review. Can you check the data flow again? I  have already implemented that. The ones that end with 'mon' are using Digimon id already.

Reply:
Please kindly show me in the data.sql file where we can find this, it's surprising I can't find it in the data.sql file
![Screenshot from 2022-09-14 14-48-23](https://user-images.githubusercontent.com/46388113/190187890-abd02edb-2918-46d5-93cb-3132216f9270.png)
Here is your data.sql file, kindly show me which line number i can find it

Reply:
Oh! I did not know it is supposed to be in data.sql
It was inside query before. I have corrected it. Thank you.